bentoml/repository/__init__.py

```diff
@@ -135,7 +136,7 @@ def _get_object_name(self, bento_name, bento_version):
     def add(self, bento_name, bento_version):
         # Generate pre-signed s3 path for upload
 
-        object_name = self._get_object_name(bento_name, bento_version)
+        object_name = self._get_object_name(bento_name, bento_version) + '.tar.gz'
```

Comment:
did you test this by manually tar zipping the BentoML bundle directory?

Reply:
Yes tested.

![image](https://user-images.githubusercontent.com/1264873/67444175-8a47ad00-f63a-11e9-8156-aa50852437ca.png)